File: curso-python/ex039.py
from datetime import datetime
#faça um programa que 

# leia o ano de nascimento de um jovem e informe de acordo com sua idade
ano_nascimento = int(input("qual o seu ano de nascimento?: "))
ano_atual = datetime.now().year
idade = ano_atual - ano_nascimento


# se já é a hora de se alistar
if idade == 18:
    print(f"você tem {idade} anos e está na idade certa para se alistar, vá a uma junta militar mais próxima!")
# Se ele ainda vai se alistar ao serviço militar
elif idade < 18:
    print(f"você tem {idade} anos e ainda não pode se alistar")
# se já passou do tempo de alistamento
else:
    print(f"você tem {idade} anos e já passou o seu processo de alistamento")


# seu programa também deverá mostrar o tempo que falta ou que passou do prazo

File: javascript-exercicios/ex039.js
const prompt = require("prompt-sync")()

let ano_nascimento = Number(prompt("qual é seu ano de nascimento?: "))
let ano_atual = new Date().getFullYear()
idade = ano_atual - ano_nascimento

if (idade == 18){
    console.log(`você tem ${idade} anos e está na idade certa para se alistar, vá a uma junta militar mais próxima!`)
}else if (idade < 18){
    console.log(`você tem ${idade} anos e você ainda não pode se alistar`)
}else{
    console.log(`você tem ${idade} anos e já passou o seu processo de alistamento`)
}
